docs/tasks/cli.md

```diff
@@ -0,0 +1,42 @@
+# Task cli
+The ```cli``` task provides an implementation for running CLI commands on
+network devices that is platform agnostic. The ```cli``` task accepts a
+command and will attempt to execute that command on the remote device returning
+the command ouput.
+
+If the ```parser``` argument is provided, the output from the command will be
+passed through the parser and returned as JSON facts using the ```engine```
+argument.
+
+
+## Requirements
+The following is the list of requirements for using the this task:
+
+* Ansible 2.5 or later
+* Connection ```network_cli```
+* ansible_network_os
+
```

Comment:
Please add a 'How to use` section that explains how to implement this task in a playbook. Try to be as verbose as possible with one (or more) examples.

Reply:
Address here https://github.com/ansible-network/network-engine/pull/93/commits/c31f18ab373dc2ce8a6f5e14e383f39c9afa9347